sample/src/main/java/io/ona/kujaku/sample/activities/MainActivity.java

```diff
@@ -22,21 +23,31 @@
 import com.android.volley.VolleyError;
 import com.mapbox.mapboxsdk.geometry.LatLng;
 
-import java.io.BufferedInputStream;
-import java.io.ByteArrayOutputStream;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.ArrayList;
+import java.util.List;
 
-import io.ona.kujaku.activities.MapActivity;
+import io.ona.kujaku.KujakuLibrary;
+import io.ona.kujaku.callables.AsyncTaskCallable;
+import io.ona.kujaku.domain.Point;
 import io.ona.kujaku.helpers.MapBoxStyleStorage;
 import io.ona.kujaku.helpers.MapBoxWebServiceApi;
+import io.ona.kujaku.listeners.OnFinishedListener;
 import io.ona.kujaku.sample.BuildConfig;
+import io.ona.kujaku.sample.MyApplication;
 import io.ona.kujaku.sample.R;
 import io.ona.kujaku.services.MapboxOfflineDownloaderService;
+import io.ona.kujaku.tasks.GenericAsyncTask;
 import io.ona.kujaku.utils.Constants;
 import io.ona.kujaku.utils.Permissions;
 
+import static io.ona.kujaku.utils.Constants.MAP_ACTIVITY_REQUEST_CODE;
+import static io.ona.kujaku.utils.Constants.NEW_FEATURE_POINTS_JSON;
+
 public class MainActivity extends BaseNavigationDrawerActivity {
 
```

Comment:
I can't see the points that I dropped previously when I opened the MapActivity and added points. Is this the desired behaviour?

If I open the MapActivity again, I can't see the points

Reply:
👍  so if I add points for the first time, re-opening the activity shows all the points previously dropped. However, subsequent point drops after clicking done does not show the new points. I have to force close the app for me to see any new points after the first point-drop session

Reply:
Yeap, this is expected. Right now, the points are fetched once when launching the MapActivity the first time.

I had added as a TODO, the need to figure out how to avoid fetching all points again and instead update only the delta.

I have added a temporary fix for this which does not take into account new points added via sync to the host app's db.